org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/preferences/cleanup/JavaFeatureTabPage.java

```diff
@@ -125,7 +125,9 @@ protected void doCreatePreferences(final Composite composite, final int numColum
 		intent(java1d7Group);
 		CheckboxPreference systemconstantsFileEncoding= createCheckboxPref(java1d7Group, 1, CleanUpMessages.JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_FileEncoding, CleanUpConstants.CONSTANTS_FOR_SYSTEM_PROPERTY_FILE_ENCODING, CleanUpModifyDialog.FALSE_TRUE);
 		CheckboxPreference systemconstantsBooleanProperty= createCheckboxPref(java1d7Group, 1, CleanUpMessages.JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_BooleanProperty, CleanUpConstants.CONSTANTS_FOR_SYSTEM_PROPERTY_BOOLEAN, CleanUpModifyDialog.FALSE_TRUE);
-		registerSlavePreference(systemconstants, new CheckboxPreference[] {systemconstantsFileSeparator,systemconstantsPathSeparator,systemconstantsLineSeparator,systemconstantsFileEncoding,systemconstantsBooleanProperty});
+		CheckboxPreference systemconstantsIntegerProperty= createCheckboxPref(java1d7Group, 1, CleanUpMessages.JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_IntegerProperty, CleanUpConstants.CONSTANTS_FOR_SYSTEM_PROPERTY_INTEGER, CleanUpModifyDialog.FALSE_TRUE);
+		CheckboxPreference systemconstantsLongProperty= createCheckboxPref(java1d7Group, 1, CleanUpMessages.JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_LongProperty, CleanUpConstants.CONSTANTS_FOR_SYSTEM_PROPERTY_LONG, CleanUpModifyDialog.FALSE_TRUE);
```

Comment:
You need to add an intent() call before the Long property checkbox is created as it gets pushed left-justified and makes the rest of the check-boxes look incorrect.  Other than that, it looks good.